src/PackageManagement.VisualStudio/ProjectSystems/VSMSBuildNuGetProjectSystem.cs

```diff
@@ -197,12 +197,13 @@ private Task AddFileCoreAsync(string path, Action addFile)
             // One exception is the 'packages.config' file, in which case we want to include
             // it into the project.
             // Other exceptions are 'web.config' and 'app.config'
+            var fileName = Path.GetFileName(path);
             if (File.Exists(Path.Combine(ProjectFullPath, path))
                 && !fileExistsInProject
-                && !path.Equals(Constants.PackageReferenceFile)
-                && !path.Equals("packages." + ProjectName + ".config")
-                && !path.Equals(EnvDTEProjectUtility.WebConfig)
-                && !path.Equals(EnvDTEProjectUtility.AppConfig))
+                && !fileName.Equals(Constants.PackageReferenceFile)
+                && !fileName.Equals("packages." + ProjectName + ".config")
+                && !fileName.Equals(EnvDTEProjectUtility.WebConfig)
+                && !fileName.Equals(EnvDTEProjectUtility.AppConfig))
```

Comment:
Since `path` can be full path, not just the file name, those file name comparisons need to be fixed.